Microsoft.Azure.Cosmos/src/Serializer/CosmosSerializer.cs

```diff
@@ -16,18 +16,18 @@ public abstract class CosmosSerializer
         /// The implementation is responsible for Disposing of the stream,
         /// including when an exception is thrown, to avoid memory leaks.
         /// </summary>
-        /// <typeparam name="T">Any typed passed to <see cref="Container"/></typeparam>
-        /// <param name="stream">The Stream response containing JSON from Cosmos</param>
+        /// <typeparam name="T">Any type passed to <see cref="Container"/>.</typeparam>
+        /// <param name="stream">The Stream response containing JSON from Cosmos DB.</param>
         /// <returns>The object deserialized from the stream.</returns>
         public abstract T FromStream<T>(Stream stream);
 
         /// <summary>
         /// Convert the object to a Stream. 
         /// The caller will take ownership of the stream and ensure it is correctly disposed of.
-        /// Stream.CanRead must be true https://docs.microsoft.com/dotnet/api/system.io.stream.canread?view=netcore-2.0
+        /// <see href="https://docs.microsoft.com/dotnet/api/system.io.stream.canread?view=netcore-2.0">Stream.CanRead</see> must be true.
```

Comment:
I would remove the `view` as it is pointing to NET Core 2 only.
```suggestion
        /// <see href="https://docs.microsoft.com/dotnet/api/system.io.stream.canread">Stream.CanRead</see> must be true.
```